brewtils/rest/system_client.py

```diff
@@ -203,17 +203,36 @@ def __init__(self, *args, **kwargs):
             )
             kwargs.setdefault("system_name", args[2])
 
-        self._system_name = kwargs.get("system_name")
-        self._version_constraint = kwargs.get("version_constraint", "latest")
-        self._default_instance = kwargs.get("default_instance", "default")
+        system_kwargs = (
+            "system_name",
+            "version_constraint",
+            "default_instance",
+            "system_namespace",
+        )
+        provided_system_kwargs = False
+        for arg in system_kwargs:
+            if kwargs.get(arg, None):
+                provided_system_kwargs = True
+                break
+
+        if provided_system_kwargs:
+            self._system_name = kwargs.get("system_name")
+            self._version_constraint = kwargs.get("version_constraint", "latest")
+            self._default_instance = kwargs.get("default_instance", "default")
+            self._system_namespace = kwargs.get(
+                "system_namespace", brewtils.plugin.CONFIG.namespace or ""
+            )
+        else:
+            self._system_name = brewtils.plugin.CONFIG.name or None
+            self._version_constraint = brewtils.plugin.CONFIG.version or "latest"
+            self._default_instance = brewtils.plugin.CONFIG.instance_name or "default"
```

Comment:
Change default values for `_version_constraint` and `_default_instance` to be `None`